src/common/resource-set-quotas.js

```diff
@@ -82,52 +81,45 @@ export default class ResourceSetQuotas extends Component {
       },
     ]
     return (
-      <Card>
-        {header !== undefined && <CardHeader>{header}</CardHeader>}
-        <CardBlock>
-          <Container>
```

Comment:
Bootstrap requires the `Container` in its [grid system](http://v4-alpha.getbootstrap.com/layout/grid/).